app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterClaimController.scala

```diff
@@ -663,12 +229,44 @@ object EnterClaimController {
         .verifying("invalid.claim", a => a.amount <= paidAmount)
     )
 
-  val entryClaimAmountForm: Form[AmountPair] = Form(
-    mapping(
-      "enter-claim.paid-amount"  -> moneyMapping(13, 2, "paid-amount.error.invalid"),
-      "enter-claim.claim-amount" -> moneyMapping(13, 2, "claim-amount.error.invalid")
-    )(AmountPair.apply)(AmountPair.unapply)
-      .verifying("invalid.claim", a => a.claimAmount <= a.paidAmount)
+  def generateClaimsFromDuties(fillingOutClaim: FillingOutClaim): Either[Error, List[Claim]] = {
```

Comment:
it would be better to have this take a `List[SelectDuties]` rather than `FillingOutClaim` - it makes testing simpler as well.

Reply:
it would be good to have a unit test for: generateClaimsFromDuties

Reply:
Done